packages/components/src/interfaces/internal/CollectionCard.ts

```diff
@@ -3,6 +3,8 @@ import type { IsomerSiteProps, LinkComponentType } from "~/types"
 
 export interface Tag {
   label: string
+  category: string
+  values: string[]
```

Comment:
yea, this is taken at build time. you can see this at `publishing/index.ts` where we write the `tag` to the sitemap

Reply:
Hmm yea, similar thoughts. I didn't think we will need the `values` array here. In the long term, this will be moved to the collection level when it will be required.

However, for current implementation, values will not be known till build time

Reply:
correct me if im wrong, but will not knowing the values till build time be an issue? 

seems like we only require all values for the filters and options, which is fine to only be known during build time

Reply:
yea, you're right - i thought we were gonna ask the users for the set of values then display them in the filters (eg: a: 0) but actually we'll hide those wtihout any matches so i'll remove this

Reply:
@adriangohjw - not sure i understand your question! i think we don't require the values at all - i think, for now, we can just fetch all values from db (collections are 1 level deep), which shuold still be manageable

Reply:
updated!